demo_classification_tf.py

```diff
@@ -0,0 +1,53 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
+import argparse
+
+def sigmoid(x):
+  return 1 / (1 + np.exp(-x))
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+        return graph_def
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--graph', help='.pb graph path', default = 'resnet_v2_101_299_frozen.pb')
+    parser.add_argument('--image', help='image path', default = 'realistic-blue-umbrella_1284-11412.jpg')
```

Comment:
There is no such image in the repository. Please remove default name and add `required=True`